owasp-top10-2017-apps/a1/copy-n-paste/app/util/db.go

```diff
@@ -88,14 +94,17 @@ func NewUser(user string, pass string, passcheck string) (bool, error) {
 	}
 	defer dbConn.Close()
 
-	query := fmt.Sprint("insert into Users (username, password) values ('" + user + "', '" + passHash + "')")
-	rows, err := dbConn.Query(query)
+	Stmt, err := dbConn.Prepare("insert into Users (username, password) values (?, ?)")
 	if err != nil {
-		return false, err
+		log.Fatal(err)
 	}
-	defer rows.Close()
+	defer Stmt.Close()
 
-	fmt.Println("User created: ", user)
+	rows, err := Stmt.Query(user, passHash)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println("User created: ", rows)
```

Comment:
I think a `defer rows.Close()` is missing in this function. Would you mind confirming this, please? 😃